cmd/dosa/main_test.go

```diff
@@ -57,11 +57,10 @@ func TestHostOptionButNothingElse(t *testing.T) {
 func TestInvalidHost(t *testing.T) {
 	c := StartCapture()
 	exit = func(r int) {}
-	os.Args = []string{"dosa", "--host", "invalid-hostname.", "schema", "check", "--prefix", "foo", "../../testentity"}
+	os.Args = []string{"dosa", "--host", "", "schema", "check", "--prefix", "foo", "../../testentity"}
```

Comment:
We'd have to add code that checks the validity of the hostname because that code no longer exists in YARPC. This is just a side-effect of a dependency update.